api/dp-openapi.yaml

```diff
@@ -733,6 +845,63 @@ components:
       in: header
       name: X-API-KEY
 
+  parameters:
+    projectid:
+      name: projectid
+      in: path
+      description: ID of project
+      required: true
+      schema:
+        type: string
+
+    projectstate:
+      name: projstate
```

Comment:
In the `project` API this is `state` and I suggest we keep that for consistency.

Reply:
done this